meta-lmp-base/recipes-extended/ostree/ostree/0007-sort-key.patch

```diff
@@ -0,0 +1,46 @@
+From 17b91cfdf0299535dcb82e3e4f2201c0bd77d6db Mon Sep 17 00:00:00 2001
+From: Jorge Ramirez-Ortiz <jorge@foundries.io>
+Date: Tue, 12 Nov 2024 21:32:36 +0100
+Subject: [PATCH] systemd-boot display order
+
+Use sort-key to display ostree deployments at the top. Other factors
+might alter the position but they should always be above the secure boot
+uefi EFI applications.
+
+Signed-off-by: Jorge Ramirez-Ortiz <jorge@foundries.io>
```

Comment:
looks like `Upstream-status` is missing

Reply:
@angolini  yes: should be 
`Upstream-Status: Inappropriate [lmp specific] `

how do I update it? or is this something you can do?

Reply:
updated